### ISSUE TITLE
Parametric objects direction argument doesn't make sense

### ISSUE THREAD

**Issue by banesullivan** [pyvista/pyvista#481]
In the following example, the direction argument is handled very strangly... I would figure that the "hills" should be oriented in the X direction if passing diretion as `(1,0,0)`.

Is this only weird for the parametric hills? what about other datasets?

```py
import pyvista as pv
mesh = pv.ParametricRandomHills(direction=(1,0,0))
mesh.plot(show_grid=True)
```

![download](https://user-images.githubusercontent.com/22067021/69785915-81388700-1176-11ea-8ce0-88b868f297cc.png)

Then, I'd expect the hills to be in the Z direction with this:
```py
mesh = pv.ParametricRandomHills(direction=(0,0,1))
mesh.plot(show_grid=True)
```

![download](https://user-images.githubusercontent.com/22067021/69785963-a4633680-1176-11ea-8afc-0f4b942e4f47.png)

**Comment by akaszynski** [comment]
Orientation isn't consistent on VTK's side.  If you look at `geometric_objects.py` you'll find that some of the transformations are modified based on the default VTK direction/orientation.

**Comment by banesullivan** [comment]
Interesting. So maybe we should just change the random hills method?

**Comment by akaszynski** [comment]
Yes.  For example, in `Cylinder` I've pre-rotated it so it is already aligned in the +Z direction:

https://github.com/pyvista/pyvista/blob/b8c53c2e30c876bd3cd29fd3f5763d49eb46aa46/pyvista/utilities/geometric_objects.py#L98-L106

Specifically, I'm referring to `surf.rotate_z(-90)`.

**Comment by banesullivan** [comment]
Ah okay. I'm going to do the same for the hills as they should be pointing in the z direction by default
